<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Execute</class>
 <widget class="QWidget" name="Execute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Execute Session</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="KUrlLabel" name="SessionURL">
       <property name="text">
        <string>Step 1: Session Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlLabel" name="ObservationsURL">
       <property name="text">
        <string>Step 2: Observations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KListWidget" name="Target"/>
     </item>
     <item>
      <widget class="KUrlLabel" name="AddObject">
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string>Add new object to list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Session">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KTitleWidget" name="ktitlewidget">
           <property name="text">
            <string>Enter Session Details:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Location:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="Location">
             <property name="text">
              <string>set location</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Begin:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KDateTimeWidget" name="Begin"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Weather:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KTextEdit" name="Weather">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Equipment:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KTextEdit" name="Equipment">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Comments:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KTextEdit" name="Comment">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="KLineEdit" name="Language"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TargetDetails">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="KTitleWidget" name="ktitlewidget_2">
           <property name="text">
            <string>View Object Details:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Magnitude:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="Mag">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="KLineEdit" name="Type">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Scheduled Time:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="SchTime">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Right Ascension:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="RA">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Declination:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="KLineEdit" name="Dec">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Altitude:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KLineEdit" name="Alt">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Azimuth:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="KLineEdit" name="Az">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="KTitleWidget" name="ktitlewidget_4">
           <property name="text">
            <string>Set observing notes for the object:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="KTextEdit" name="Notes">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="Slew">
             <property name="text">
              <string comment="Move the telescope to an object or location">Slew Telescope</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="KTitleWidget" name="ktitlewidget_3">
           <property name="text">
            <string>Enter the Observation Details:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Observer</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KComboBox" name="Observer"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Telescope:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KComboBox" name="Scope"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Eyepiece:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KComboBox" name="Eyepiece"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Lens:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KComboBox" name="Lens"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KComboBox" name="Filter"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Seeing:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="KDoubleNumInput" name="Seeing"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>arc seconds</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Faintest Star:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="KDoubleNumInput" name="FaintestStar"/>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>(magnitude)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QTimeEdit" name="Time"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="KTextEdit" name="Description"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="NextButton">
         <property name="text">
          <string>Next &gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>kdatetimewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
